infra/network/src/test/java/com/simprints/infra/network/apiclient/SimApiClientImplTest.kt

```diff
@@ -59,7 +56,7 @@ class SimApiClientImplTest {
 
     @Test
     fun `should throw a backend maintenance exception without estimated outage when it's a maintenance error without header and not retry`() =
-        runTest(StandardTestDispatcher()) {
+        runTest {
```

Comment:
Default is UnconfinedTestDispatcher!?

Reply:
From what I gathered in sources, it is `StandardTestDispatcher` by default, meaning this declaration was redundant.

Reply:
Oh, my bad!